less/navigation.less

```diff
@@ -1 +1,46 @@
-// Navigation code here
\ No newline at end of file
+// main: index.less
+// Navigation code here
+
+nav {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    width: 85%;
```

Comment:
I would try the width at 100%? I tried it and it looks great for mobile and tablet view!